packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -341,10 +336,10 @@ class ConnectionStateHandler implements IConnectionStateHandler {
                 category: source === "timeout" ? "error" : "generic",
                 details: JSON.stringify({
                     source,
-                    pendingClientId: this.pendingClientId,
-                    clientId: this.clientId,
+                    clientId: this.pendingClientId,
```

Comment:
But pendingClientId is different from clientId, we should log it as such.

Reply:
Changed it back